### PR TITLE
Input parsing

### PR DESCRIPTION
**Pull request by jpinot**
Create new Class InputParse:
- Get puzzle from FD.
- Check if errors.
- Translate puzzle.
- Translate puzzle back.

### CODE REVIEW

**Review by jpinot — COMMENTED** [review]
Thing of better name for some methods?!.

File: includes/inputParser.h [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	private:
+		vector<unsigned char>	puzzle_;
+		InputErrors		errors_;
+		unsigned char		size_;
+		unsigned char		maxValidTile_;
+		vector<bool>		tileChecker_;
+
+		void		setTileChecker();
+		const bool	tileIsValid(const unsigned char& tile) const {return (tile < maxValidTile_ && !tileChecker_[tile]);}
+		void		setTileAsChecket(const unsigned char& pos) {tileChecker_.at(pos) = true;}
```
Comment (mfiguera) [inline includes/inputParser.h]: Rename -> fix typo setTileAsChecked

**Review by mfiguera — CHANGES_REQUESTED** [review]
Fix typo, everything else seems good

File: includes/inputParser.h (hunk 1 above)
Comment (jpinot) [reply includes/inputParser.h]: commit 1570316

**Pull request merged**